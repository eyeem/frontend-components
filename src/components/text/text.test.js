import React from 'react';
import { render, cleanup, getByText } from 'react-testing-library';

import Text from './index';

afterEach(cleanup);

test('it render default text', () => {
  const { container } = render(<Text>Hello World</Text>);
  const hasRenderedChildText = getByText(container, 'Hello World');
  const component = container.firstChild;
  expect(hasRenderedChildText).toBeTruthy();
  expect(component).toMatchSnapshot();
});
