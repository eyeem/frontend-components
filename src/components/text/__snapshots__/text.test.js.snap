// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it render default text 1`] = `
.c0 {
  display: block;
  -webkit-letter-spacing: 0.4px;
  -moz-letter-spacing: 0.4px;
  -ms-letter-spacing: 0.4px;
  letter-spacing: 0.4px;
  text-align: left;
  color: #333333;
  fill: #333333;
  font-size: 14px;
  line-height: 22px;
  -webkit-text-decoration: none;
  text-decoration: none;
  margin: 0;
  margin-bottom: 11px;
  margin-top: 0px;
  -webkit-transition: color 0.2s ease-out;
  transition: color 0.2s ease-out;
  font-weight: 300;
  font-family: "Sailec Regular",sans-serif;
}

.c0:focus,
.c0:hover {
  color: #333333;
  fill: #333333;
}

.c0:active {
  color: #333333;
  fill: #333333;
}

.c0:focus {
  outline: none;
}

.c0:last-child {
  margin-bottom: 0;
}

@media (min-width:40.0625em) {
  .c0 {
    font-size: 16px;
    line-height: 24px;
    margin-bottom: 12px;
    margin-top: 0px;
  }
}

@media (min-width:48.0625em) {
  .c0 {
    font-size: 16px;
    line-height: 24px;
    margin-bottom: 12px;
    margin-top: 0px;
  }
}

@media (min-width:64.0625em) {
  .c0 {
    font-size: 16px;
    line-height: 24px;
    margin-bottom: 12px;
    margin-top: 0px;
  }
}

<p
  class="c0"
>
  Hello World
</p>
`;
